### PR TITLE
Use cron for labeler trigger

### PR DESCRIPTION
**Pull request by damccorm**
We can't give write permissions to forks, so we want to just run this from the default branch on a recurring schedule

### CODE REVIEW

**Comment by drivebikesss** [comment]
6898b66

**Comment by SynovialWheat9011** [comment]
# 12

**Comment by DrZainAzmi** [comment]
TRANSFER from 0x0000000000000000000000000000000000000000 to 0xd0322afa71bc9099cc0004aff056f4fb31bd6e98 8,160.2338 ETH 

[https-etherscan.io-address-0x0000000000000000000000000000000000000000.pdf](https://github.com/actions/starter-workflows/files/4457590/https-etherscan.io-address-0x0000000000000000000000000000000000000000.pdf)

File: automation/label.yml [hunk 1]
```diff
@@ -9,4 +11,4 @@ jobs:
     steps:
     - uses: actions/labeler@v2
```
Comment (iAMnormztwizedumz) [inline automation/label.yml:12]: Labeler V2, capital as described the Person, subject under person small or (), eTc. As per agreement Person state must be CaPiTaL. (CPTL) in a Manner oF senTeNcE grammar and codes used sucHLikE gitHub codes. SamE aS I wriTE, before I looked untO diveRsion thAts how I typE, nOW small cap follOwEd bY big caps unto MucH cLEareR DirecTivEs.

Comment (iAMnormztwizedumz) [inline automation/label.yml:12]: Labeler V2, capital as described the Person, subject under person small or (), eTc. As per agreement Person state must be CaPiTaL. (CPTL) in a Manner oF senTeNcE grammar and codes used sucHLikE gitHub codes. SamE aS I wriTE, before I looked untO diveRsion thAts how I typE, nOW small cap follOwEd bY big caps unto MucH cLEareR DirecTivEs.

Comment (iAMnormztwizedumz) [inline automation/label.yml:12]: Labeler V2, capital as described the Person, subject under person small or (), eTc. As per agreement Person state must be CaPiTaL. (CPTL) in a Manner oF senTeNcE grammar and codes used sucHLikE gitHub codes. SamE aS I wriTE, before I looked untO diveRsion thAts how I typE, nOW small cap follOwEd bY big caps unto MucH cLEareR DirecTivEs.

Comment (iAMnormztwizedumz) [inline automation/label.yml:12]: Labeler V2, capital as described the Person, subject under person small or (), eTc. As per agreement Person state must be CaPiTaL. (CPTL) in a Manner oF senTeNcE grammar and codes used sucHLikE gitHub codes. SamE aS I wriTE, before I looked untO diveRsion thAts how I typE, nOW small cap follOwEd bY big caps unto MucH cLEareR DirecTivEs.

**Pull request closed without merging**